packages/gatsby/src/db/nodes.js

```diff
@@ -23,31 +23,32 @@ interface NodeStore {
 /**
  * Get content for a node from the plugin that created it.
  *
- * @param {Object} node
- * @returns {promise}
+ * @param {IGatsbyNode} node
+ * @returns {Promise<string>}
  */
-function loadNodeContent(node) {
+async function loadNodeContent(node) {
   if (_.isString(node.internal.content)) {
     return Promise.resolve(node.internal.content)
```

Comment:
do we need to still wrap it in `Promise.resolve` if function is `async`?

Reply:
No :)